modules/core/24-host/channel_keys.go

```diff
@@ -52,3 +54,8 @@ func ChannelRestoreKey(portID, channelID string) []byte {
 func channelPath(portID, channelID string) string {
 	return fmt.Sprintf("%s/%s/%s/%s", KeyPortPrefix, portID, KeyChannelPrefix, channelID)
 }
+
+// ErrorPath stores the ErrorReceipt in the case that a chain does not accept the proposed upgrade
+func ErrorPath(portID, channelID string) string {
```

Comment:
Could we rename to `ChannelUpgradeErrorPath`, I think it would be best to be explicit. Also another mega nit but could we add the key and path funcs above the `channelPath` private func such that all exported funcs read first in the file